pydoctor/model.py

```diff
@@ -808,6 +808,8 @@ def introspectModule(self,
             module_full_name = f'{package.fullName()}.{module_name}'
 
         spec = importlib.util.spec_from_file_location(module_full_name, path)
+        if spec is None: 
+            raise RuntimeError(f"Cannot find spec for module {module_full_name} at {path}")
```

Comment:
Can you do a test for this apparently untested case in a follow-up?

Reply:
Like @mthuurne was saying, this error is raised when  a file is deleted or moved between the moment its name is discovered and the moment it is introspected. So honestly I don't know how to test such a case.

Reply:
This is why `FilePath` narrows down its interface to a single object which returns other objects, so you can pass a parameter along to the system under test. In the absence of such compositional design, the way to test this is to mock out `spec_from_file_location` with `patch` in a test.